packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmDictionaryTests.kt

```diff
@@ -1959,6 +2026,296 @@ internal abstract class ManagedDictionaryTester<T>(
         }
     }
 
+    override fun containsKey() {
+        val dataSet = typeSafetyManager.dataSetToLoad
+
+        errorCatcher {
+            realm.writeBlocking {
+                val dictionary = typeSafetyManager.createContainerAndGetCollection(this)
+                dictionary.putAll(dataSet)
+
+                dataSet.forEach {
```

Comment:
Same as in other PR's. Not 100% sure why we are duplicating the asserts between the `errorCatcher` and `assertContainerAndCleanup`.